model_compression_toolkit/core/tpc_models/default_tpc/v4/tpc_keras.py

```diff
@@ -75,8 +75,9 @@ def generate_keras_tpc(name: str, tp_model: tp.TargetPlatformModel):
         tp.OperationsSetToLayers("FullyConnected", [Dense])
         tp.OperationsSetToLayers("AnyReLU", [tf.nn.relu,
                                              tf.nn.relu6,
-                                             tp.LayerFilterParams(ReLU, negative_slope=0.0),
-                                             tp.LayerFilterParams(Activation, activation="relu")])
+                                             ReLU,
```

Comment:
Please add also "tf.nn.leaky_relu" and "tf.keras.layers.LeakyReLU" to the list (for all the files)